nilearn/maskers/multi_nifti_masker.py

```diff
@@ -176,6 +191,7 @@ class MultiNiftiMasker(NiftiMasker):
     def __init__(
         self,
         mask_img=None,
+        runs=None,
```

Comment:
change in API but this parameter exists in the parent class
so to me this is a bug fix more than an enhancement because users could have done